styles/trip-house-styles.css

```diff
@@ -39,3 +40,58 @@
 	font-size: 24px;
 	color: #383838;
 }
+
+/*footer-section*/
+
+.footer {
+	height: 625px;
```

Comment:
Don't use fixed height for block. Height must be achieved with content and gaps(margin, padding, etc.)